src/main/resources/static/css/typography.css

```diff
@@ -59,3 +59,25 @@ div.active a {
 a.link-large {
     font-size: large;
 }
+
+.custom-select-size {
+    font-size: 1.2rem;
+    text-align-last: center;
+}
+
+.custom-btn {
+    font-size: 25px;
+}
+
+#subjectModalLabel {
+    margin-left: 10px;
+}
+
+#errorModalLabel {
+    text-align: center;
+    margin-left: 184px;
```

Comment:
what do you actually want to do here? put it in the center?

Reply:
yes to center "warning", text-align from css and text-center from bootstrap didn't work so I used margin left

Reply:
We never center something with a margin. Try making your browser window smaller and you will see why. Try googling the problem that it won't center. (hint: probably some problem with display:block)

Reply:
managed